### PR TITLE
Feat: Button Component

### PR DESCRIPTION
**Pull request by lomesshh**

### CODE REVIEW

**Comment by Shah-Arsalan** [comment]
Provide utility class for repeating color

**Comment by lomesshh** [comment]
> Provide utility class for repeating color

as you suggested I have added utility class for white-color

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+.btn-outline-primary{
+    background-color: transparent;
+    color: var(--primary-color);
+    border: 1px solid var(--primary-color);
+}
+
+.btn-outline-success{
+    background-color: transparent;
+    color: var(--success-color);
+    border: 1px solid var(--success-color);
```
Comment (anuj-kumary) [inline components/button.css:45]: Use utility class for border width and style or add in base class

Comment (anuj-kumary) [inline components/button.css:45]: Use utility class for border width and style or add in base class

Comment (lomesshh) [reply components/button.css:45]: I have made changes to the code regarding the same, which will reflect in my upcoming PR.

**Pull request merged**